Formula/busco.rb

```diff
@@ -15,17 +15,57 @@ class Busco < Formula
     sha256 "f77caa27c9ea8e866682a00cd93bbb626f6139b6a95e1c7354b0268ad806a253" => :x86_64_linux
   end
 
+  depends_on "augustus"
+  depends_on "hmmer"
   depends_on "python"
   depends_on "blast" => :recommended
-  # Also depends on augustus and hmmer
+  depends_on "r" => :optional
```

Comment:
```suggestion
```
I'd prefer not to add options to Brewsci/bio unless there's a strong motivation.
On macOS the `r` cask is much preferred over the `r` formula.

Reply:
I think in this case it would be best to remove the `:recommended` from blast. There's no real reason for this.

Reply:
Agreed.